### DISCUSSION TITLE
A chance to shape the future of Green Software Patterns

### DISCUSSION THREAD

**Discussion by russelltrow** [Green-Software-Foundation/patterns#280] [Announcements]
Are you passionate about sustainable software and part of a member organization of the Green Software Foundation (GSF)? We're looking for enthusiastic and dedicated practitioners like you to make a meaningful contribution to improving green software patterns.

This opportunity is exclusive to members of the Green Software Foundation. [Check if your organization is a member](https://greensoftware.foundation/team)

## The Opportunity
We are looking for people to participate in a **series of virtual Design Thinking workshops** to shape the future of our [Green Software Patterns](https://grnsft.org/wiki-pr-patterns). We currently have a catalogue of 50 patterns covering AI, Cloud & Web which have been adopted by a number of organisations. 

We now wish to evolve the catalogue, making it relevant to a larger audience, covering additional areas and further establishing it as an authority on patterns to reduce your software emissions.

### We're seeking individuals who have
* A passion for sustainable software development
* Background in any area of software - engineering, architecture, product, user experience design
* At least basic experience in sustainable software development
* Ability to commit up to 12 hours over 3-4 weeks

### We need you to help us
* Answer “What’s next for Patterns?”
* Ensure our catalogue is valuable to the widest possible audience
* Develop a product roadmap and plan of action

## Register your interest
We will be running the workshops virtually in January & February 2024, the exact dates will be agreed with the interested parties.

To register your interest simply post a comment below!

## Workshop Details

Details of the workshop can be found on the project wiki: https://github.com/Green-Software-Foundation/patterns/wiki/Patterns-Strategy-workshops-%E2%80%90-Jan-2024

The workshops will run on:

* Wednesday 24 January 2024
* Wednesday 31 January 2024
* Wednesday 7 February 2024
* Wednesday 14 February 2024

**Comment by greenhsu123** [comment]
Interested!

**Comment by pdematos12** [comment]
Interested - but am fairly new to GSF and patterns. I can contribute a UX perspective.

**Comment by sk16-dev** [comment]
Interested! New to GSF as of this autumn. Can contribute a product operations and management perspective.

**Comment by cdhaens** [comment]
We are interested - Carolin (Siemens)

**Comment by russelltrow** [comment]
@kapokasa, @franziska-warncke would you be interested in participating?

@danielvaughan would someone from your team (Sebastien?) like to be involved?

@filga would you or someone from your team like to be involved?

**Comment by Cavvar** [comment]
Hi!
I am interested - Denis(NTT)

**Reply by russelltrow** [reply]
Hi Denis, thank you so much for expressing your interest. Could I please ask that you complete https://greensoftware.foundation/onboarding/ so I can get your email and add you to the proceedings.

**Reply by Cavvar** [reply]
Hi @russelltrow,

I have registered as suggested. Thanks for letting me join the workshop!

**Reply by russelltrow** [reply]
Great to have you onboard. I've added you to the mailing list and calendar invites.

**Comment by filga** [comment]
Hi @russelltrow 
both @sheptris and I are interested in the discussion!

**Comment by russelltrow** [comment]
In this week's project call we came to consensus on the the following timings:

* First workshop Wednesday 17th January 2024
* At 15:00 GMT
* 2 hours long
* Further workshops at the same time in the following weeks

**Comment by muemich** [comment]
Could you please invite me as well? Thanks

**Comment by navveenb** [comment]
Interested

**Comment by kapokasa** [comment]
Interested!

**Comment by franziska-warncke** [comment]
Interested

**Comment by ikolaxis** [comment]
Interested

**Comment by pgregrivera** [comment]
Interested

**Comment by philippqf** [comment]
I am interested and I have a question:

> Ability to commit up to 12 hours over 3-4 weeks

Is that 12 hours per week or a total of 12 hours?

**Reply by russelltrow** [reply]
Hi @philippqf a total of 12 hours spread over a number of weeks.

**Reply by philippqf** [reply]
Sounds good. I'm in 😃

**Comment by peaMS** [comment]
interested! (Paola.annis@microsoft.com)

**Comment by sfontanarrosa** [comment]
Interested! (s.fontanarrosa@globant.com)

**Comment by tellingmachine** [comment]
I am interested in participating.

**Comment by russelltrow** [comment]
Thank you everyone who has expressed their interest in the workshop. We now have a good number of people so I'm going to close registrations for the time being.

For those who have expressed interest I will be in contact this week with further details.

**Reply by greenhsu123** [reply]
Thanks, Russ! Looking forward to it!

**Comment by imdacharlene** [comment]
Keen. charlene_wong@imda.gov.sg

**Comment by simonthurman** [comment]
I'm interested in supporting.

**Reply by russelltrow** [reply]
Hi Simon, thank you so much for expressing your interest. Could I please ask that you complete https://greensoftware.foundation/onboarding/ so I can get your email and add you to the proceedings.

**Comment by tholewis** [comment]
It appears you have enough folks, but if there is some drop off, I am glad to help support as well.

**Comment by russelltrow** [comment]
Hi all,

Today I have been coordinating with the workshop facilitator and I can confirm that the workshops will take place:

* Wednesday 24 January 2024
* Wednesday 31 January 2024
* Wednesday 7 February 2024
* Wednesday 14 February 2024

Each workshop will start at 16:00 GMT and run for 2 hours.

I will send further details and calendar invites out to everyone above on Monday.

**Comment by russelltrow** [comment]
Details about the workshops will live on the project wiki going forward: https://github.com/Green-Software-Foundation/patterns/wiki/Patterns-Strategy-workshops-%E2%80%90-Jan-2024

**Comment by simonthurman** [comment]
Thank Russell, I've completed the form.
________________________________

**Reply by russelltrow** [reply]
Great to have you onboard. I've added you to the mailing list and calendar invites.

**Comment by EvanPrice0** [comment]
interested

**Reply by russelltrow** [reply]
Fantastic news, I've added you to the mailing list and sent over information about the workshops.

**Comment by cf-almeida** [comment]
Interested as well!

**Reply by russelltrow** [reply]
Added to mailing list and calendar invites.

**Comment by KarthikeyanKumar17** [comment]
@russelltrow Interested & Registered Thanks ☺️
